### PR TITLE
ENYO-5702: Add QA sample to test navigating into an overflow container

### PR DESCRIPTION
**Pull request by haileyR**
### Issue Resolved / Feature Added
ENYO-5702: Within QA-Sampler Focus target is inconsistent with sample code

### Resolution
Added ScrollerWithSpotlightTargetCalculation sample.

### CODE REVIEW

File: packages/sampler/stories/qa/Scroller.js [hunk 1]
```diff
@@ -263,4 +283,11 @@ storiesOf('Scroller', module)
 		() => (
 			<ScrollerWithTwoExpandableList />
 		)
+	).add(
+		'With Spotlight Target Calculation',
+		() => (
+			<div>
```
Comment (viodragon2) [inline packages/sampler/stories/qa/Scroller.js]: no need for extra `<div>`

File: packages/sampler/stories/qa/Scroller.js [hunk 2]
```diff
@@ -125,6 +130,21 @@ class ScrollerWithTwoExpandableList extends React.Component {
 	}
 }
 
+class ScrollerWithSpotlightTargetCalculation extends React.Component {
+	render () {
+		return (
+			<div>
+				<Button>hello</Button>
+				<Scroller focusableScrollbar style={{height: ri.scale(300)}}>
+					<Repeater childComponent={CustomItem}>
```
Comment (viodragon2) [inline packages/sampler/stories/qa/Scroller.js]: I don't think `CustomItem` is required for demonstrating the problem.

File: packages/sampler/stories/qa/Scroller.js [hunk 3]
```diff
@@ -156,6 +158,30 @@ class ScrollerWithLargeContainer extends React.Component {
...
+	render () {
+		return (
+			<div>
+				<Panel>
+					<Header title="Scroll Intent (Scroller)">
+						<Button>hello</Button>
+					</Header>
+					<Scroller
+						focusableScrollbar
+						data-webos-voice-focused
```
Comment (viodragon2) [inline packages/sampler/stories/qa/Scroller.js]: voice prop is irrelevant to the sample

File: packages/sampler/stories/qa/Scroller.js [hunk 4]
```diff
@@ -156,6 +158,30 @@ class ScrollerWithLargeContainer extends React.Component {
 	}
 }
 
+class ScrollerWithSpotlightTargetCalculation extends React.Component {
+	render () {
+		return (
+			<div>
+				<Panel>
+					<Header title="Scroll Intent (Scroller)">
```
Comment (viodragon2) [inline packages/sampler/stories/qa/Scroller.js]: Title is irrelevant to the sample

File: packages/sampler/stories/qa/Scroller.js [hunk 5]
```diff
@@ -156,6 +158,30 @@ class ScrollerWithLargeContainer extends React.Component {
...
+						<Button>hello</Button>
+					</Header>
+					<Scroller
+						focusableScrollbar
+						data-webos-voice-focused
+					>
+						<Repeater
```
Comment (viodragon2) [inline packages/sampler/stories/qa/Scroller.js]: minor nit: Just for the cleanliness-sake for the import section of the code, let's use `Group` instead as in the other samples.

File: packages/sampler/stories/qa/Scroller.js [hunk 6]
```diff
@@ -156,6 +158,30 @@ class ScrollerWithLargeContainer extends React.Component {
 	}
 }
 
+class ScrollerWithSpotlightTargetCalculation extends React.Component {
```
Comment (viodragon2) [inline packages/sampler/stories/qa/Scroller.js]: It's an overkill to create as a separate class as it doesn't require any react lifecycle methods or state to manage. Let's move the render part to the `.add()` of stories.

File: packages/sampler/stories/qa/Scroller.js [hunk 7]
```diff
@@ -156,6 +158,30 @@ class ScrollerWithLargeContainer extends React.Component {
...
+	render () {
+		return (
+			<div>
```
Comment (viodragon2) [inline packages/sampler/stories/qa/Scroller.js]: extra div unnecessary
